### PR TITLE
Update clang_format

### PR DESCRIPTION
**Pull request by rbudai98**
Updated clang-format according to https://github.com/ament/ament_lint/blob/rolling/ament_clang_format/ament_clang_format/configuration/.clang-format

### CODE REVIEW

File: src/publisher_factory.cpp [hunk 1]
```diff
@@ -35,101 +35,93 @@
...
-                                 const std::shared_ptr<aditof::Camera> &camera,
-                                 aditof::Frame **frame) {
+void PublisherFactory::createNew(
+  ModeTypes mode, ros::NodeHandle nHandle, const std::shared_ptr<aditof::Camera> & camera,
+  aditof::Frame ** frame)
+{
+  ros::Time timeStamp = ros::Time::now();
 
-    ros::Time timeStamp = ros::Time::now();
+  if (*frame != nullptr) (*frame)->~Frame();
```
Comment (VHolonec) [inline src/publisher_factory.cpp:44]: Can you call the destructor with delete keyword?
delete *frame;

Comment (rbudai98) [reply src/publisher_factory.cpp:44]: Is there a specific reason for it?

Comment (VHolonec) [reply src/publisher_factory.cpp:44]: I usually used just delete keyword to call the destructor. Calling explicity will call the base destructor or just the called destructor?

**Review by VHolonec — APPROVED** [review]

File: src/publisher_factory.cpp (hunk 1 above)
Comment (rbudai98) [reply src/publisher_factory.cpp:44]: I see. Since frame hase no child members, for this case we might keep this form

**Pull request merged**